src/stats/boxplot.jl

```diff
@@ -124,6 +130,8 @@ function Makie.plot!(plot::BoxPlot)
                 # change q1 and q5 to show outliers
                 # using maximum and minimum values inside the limits
                 q1, q5 = extrema_nan(inside)
+                # register boxcolor
+                push!(boxcolor, getuniquevalue(plot[:color].val, idxs))
```

Comment:
```suggestion
                push!(boxcolor, getuniquevalue(color, idxs))
```

Looks good to me other than this change.

Reply:
Oops